temporalio/workflow.py

```diff
@@ -1091,8 +1166,29 @@ def uuid4() -> uuid.UUID:
     return uuid.UUID(bytes=random().getrandbits(16 * 8).to_bytes(16, "big"), version=4)
 
 
+async def sleep(
```

Comment:
May want to update README (unsure if that would be just the `Timers` section or also the `Workflow Utilities` section), but don't have to